### DISCUSSION TITLE
Any way to use this remotely (like over Cloudflare?)

### DISCUSSION THREAD

**Discussion by Gyramuur** [LostRuins/koboldcpp#503] [Q&A]
The United version has a --remote flag that allows you to host a public server via Cloudflare. However, I want to use this version of kobold, as I want to use a 20B GGUF model (doesn't seem like any GPTQ version exists), and United doesn't recognise GGUF.

It seems like this version of Kobold doesn't have an equivalent remote feature, though? Is there a way to play this with other people?

**Comment by LostRuins** [comment]
It does, you have a few options:
1. Use a remote cloudflare tunnel, included with the Remote-Link.cmd file in this repo. Simply run it, and launch Remote-Link.cmd, works on both linux and windows.
2. Port forward your local instance (not recommended)
3. Run it over horde. Koboldcpp comes with an embedded horde worker, and can share its instance over horde enabled clients like https://lite.koboldai.net
For more info, please read the wiki, it is explained in detail. https://github.com/LostRuins/koboldcpp/wiki

**Comment by Gyramuur** [comment]
Am I able to use remote cloudfare with the standalone exe that's provided on the releases page?

**Reply by LostRuins** [reply]
Hmm not yet, but I might add it as a feature

**Reply by Gyramuur** [reply]
Ah, that would be really good. I would try the cloudflare bat offered in the repo, but admittedly I don't know how to get the files in the source code to work (I'm on Windows and using the pre-released .exe as recommended). I see that the .exe has a "--host" option? Would that allow me to potentially allow other people to connect?
